packages/types/lib/driver.ts

```diff
@@ -615,6 +615,37 @@ export interface Core<C extends Constraints = BaseDriverCapConstraints> {
   isCommandsQueueEnabled: boolean;
   eventHistory: EventHistory;
   onUnexpectedShutdown(handler: () => any): void;
+  /**
+   * @summary Retrieve the server's current status.
+   * @description
+   * Returns information about whether a remote end is in a state in which it can create new sessions and can additionally include arbitrary meta information that is specific to the implementation.
+   *
+   * The readiness state is represented by the ready property of the body, which is false if an attempt to create a session at the current time would fail. However, the value true does not guarantee that a New Session command will succeed.
+   *
+   * Implementations may optionally include additional meta information as part of the body, but the top-level properties ready and message are reserved and must not be overwritten.
+   *
+   * @example
+   * ```js
+   * // webdriver.io example
+   * await driver.status();
+   * ```
+   *
+   * ```python
+   * selenium.webdriver.common.utils.is_url_connectable(port)
```

Comment:
i don't think this is right. @KazuCocoa what is `GET /status` in the python bindings?

Reply:
Maybe Python client does not have it. `is_url_connectable` actually calls the local https://github.com/SeleniumHQ/selenium/blob/084c3b16bb11d625536141bc441ec95a8ee05bb7/py/selenium/webdriver/common/utils.py#L122 but maybe it is.

Reply:
hm, actually the selenium client does not have https://github.com/SeleniumHQ/selenium/blob/086e9fb1536348a891af9a52853c53021566dd50/py/selenium/webdriver/remote/command.py#L99
https://github.com/SeleniumHQ/selenium/blob/084c3b16bb11d625536141bc441ec95a8ee05bb7/py/selenium/webdriver/remote/remote_connection.py#L182

let me create a pr as `driver.get_status()`

Reply:
I just copied the example from the website

Reply:
Yea, i saw the page.
A new version will have `driver.getStatus()` https://github.com/appium/python-client/pull/820 (will push a new version later)